web/src/challange/loader.js

```diff
@@ -46,7 +47,8 @@ export function stop(countdown, failed = false){
             clearInterval(interval);
             window.location.reload();
         }
-        else if (countdown > 0) {
+        else if (countdown > 0){
```

Comment:
In other places you seem to add a space before the `{`, but here you remove it, is it intentional? :)

Reply:
Oh that wasn't intentional. I think I will fix the gh-action to actually run eslint on CI too